lib/manageiq/api/common/request.rb

```diff
@@ -31,17 +44,24 @@ def self.with_request(request)
         end
 
         def self.current_forwardable
-          raise ManageIQ::API::Common::HeadersNotSet, "Current headers have not been set" unless current
-          current.forwardable
+          current!.forwardable
         end
 
         attr_reader :headers, :original_url
 
-        def initialize(headers:, original_url:, **kwargs)
+        def initialize(headers:, original_url:, **_kwargs)
```

Comment:
should we remove the unused `**_kwargs`?

Reply:
This is there to capture and "extra" parameters that might be sent along, otherwise the caller blows up.  We can probably remove it if we are sure all callers are doing the right thing.  I'd leave that to a separate PR, though.